pkg/ddevapp/ddevapp.go

```diff
@@ -1021,6 +1023,20 @@ func (app *DdevApp) GetDBImage() string {
 func (app *DdevApp) Start() error {
 	var err error
 
+	if app.DdevVersionConstraint != "" {
+		v, err := semver.NewVersion(versionconstants.DdevVersion)
+		if err != nil {
+			util.Warning("%v %v, ddev version constraint cannot be verified", err, versionconstants.DdevVersion)
```

Comment:
```suggestion
			util.Warning("%v %v, ddev version could not be parsed", err, versionconstants.DdevVersion)
```

Reply:
there's an error message around that coming from the server variable, I just wanted to emphasize that the intended goal was not going to work.